ts/editor/RichTextBadge.svelte

```diff
@@ -15,16 +15,19 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     $: icon = off ? richTextOff : richTextOn;
 </script>
 
-<span on:click|stopPropagation={toggle}>
+<span class:highlighted={off} on:click|stopPropagation={toggle}>
```

Comment:
Why is this highlighted when off? Shouldn't a greyed out state show that the rich text input is not visible?

Reply:
My thinking was that rich text input will be visible far more frequently than hidden. Hiding fields is an exception, not the rule. Therefore, I'd recommend graying it out when it's on to reduce visual clutter.

You can try it out the other way. I've found it to be unneccessarily distracting.

Reply:
Okay I see - makes sense.
This also leads me back to [this suggestion of mine](https://forums.ankiweb.net/t/default-editor-settings-instead-of-persistent/12761). In that case, the icon would be highlighted if it does not match the default.

Reply:
Yeah, I think that's for the best. Highlights only serve their purpose when they're in the minority.
I renamed the commit to reflect that idea better.